manifests/0000_50_olm_11-olm-operators.configmap.yaml

```diff
@@ -6,6 +6,127 @@ metadata:
 data:
   customResourceDefinitions: |-
   clusterServiceVersions: |-
```

Comment:
I changed configmap.yaml to keep both records in sync, not sure if we will run into validation error from registry otherwise.